server_src/lib/serverCommands.js

```diff
@@ -131,7 +130,9 @@ class SystemCommands {
           "<b class='text-danger'>Cancel:</b> This option will cancel the update process and not install the required dependencies. No update will run and manual intervention by the user is required. <br><br>";
         return clientResponse;
       } else {
-        await installMissingNpmDependencies();
+        for (let i = 0; i > missingPackagesList.length; i++) {
+          await installNpmDependencies(missingPackagesList[i]);
+        }
```

Comment:
I use a for loop here due to the async await

Reply:
It's checked initially to make sure it's got a length worth iterating over. If not it's the previous return user status anyway.

Reply:
`installNpmDependencies` should not be plural